### ISSUE TITLE
[FEAT] add restricted configuration `conventionalCommits.promptDescription`

### ISSUE THREAD

**Issue by yi-Xu-0100** [vivaxy/vscode-conventional-commits#189]
### Is your feature request related to a problem? Please describe

Add restricted configuration `conventionalCommits.promptDescription` to disable the `description` section. But it will only work when the `conventionalCommits.autoCommit` setting to false **or** the `conventionalCommits.showEditor` setting to true. 😁

### Describe the solution you'd like

I always edit the commit message as a text document in a separate tab. The large view is better than an input box. So I only need the choices of `type`, `scope`, and `gitmoji`, then I want to edit the main message rather than edit the description in the input box. 😊

**Comment by vivaxy** [comment]
@yi-Xu-0100 Oh, I understand what problem the `promptDescription` option is intended to resolve.

Actually, I am not quite satisfied with the extension right now. I was using https://github.com/lppedd/idea-conventional-commit lately. I find it is quite efficient when entering the commit messages in an input box, rather than selecting options like we do now. It's quite a huge change and it requires tons of work to do. I'd like to try that way.

**Comment by yi-Xu-0100** [comment]
@vivaxy In my opinion, I think the best part of this extension is that it has a good visualization process. For people without programming experience, it only needs to give hints to help complete the whole process. No need to learn how to get tips. 

No matter how long I haven't programmed again, as long as I use this extension, I can complete the whole process without more help. 😁

**Comment by vivaxy** [comment]
@yi-Xu-0100 That's right, "Commit by selecting the options" and "Commit by typing with completion" are two different kinds of experience.

https://github.com/bendera/vscode-commit-message-editor "Edit as form" is another approach of "Commit by selecting the options". It has a better experience of writing descriptions. When it comes to the efficiency, it seems to involve the mouse moving which I think is less efficient. I think they can add some hotkeys to solve the problem.

For the users who are not familiar with "Conventional Commits", "Commit by selecting the options" is better. It can give a guidance of the "Conventional Commits" format.

When users are familiar with "Conventional Commits", then tends to choose a more efficient way. I think "Commit by typing with completion" is that way. In VSCode, there is not much choice for those who prefer efficiency over guidance.

That's why I am digging into "Commit by typing with completion". I think it should be another extension for now.
